apps/schools/management/commands/create_schoolprogrammes.py

```diff
@@ -18,7 +19,11 @@
     "dharwad",
     "chitradurga",
     "chamarajanagara",
-    "bangalore rural"
+    "bangalore rural",
+
+    # Odisha
+    "rayagada",
+    "bolangir",
```

Comment:
This script maps the GKA programme to the respective schools that are a part of it. We already have the GKA districts in Karnataka. Added the two districts from Odisha.